src/systems/a320_systems_wasm/src/lib.rs

```diff
@@ -204,13 +238,13 @@ impl SimulatorAspect for Autobrakes {
         }
     }
 
-    fn pre_tick(&mut self, delta: Duration) {
+    fn pre_tick(&mut self, _delta: Duration) {
```

Comment:
When unused, prefer `_` over `_delta`, unless on the default fn implementation on a trait.
```suggestion
    fn pre_tick(&mut self, _: Duration) {
```